salt/modules/mac_service.py

```diff
@@ -150,6 +150,37 @@ def _get_service(name):
     raise CommandExecutionError('Service not found: {0}'.format(name))
 
 
+def _always_running_service(name):
+    '''
+    Check if the service should always be running based on the KeepAlive Key
+    in the service plist.
+
+    :param str name: Service label, file name, or full path
+
+    :return: True if the KeepAlive key is set to True, False if set to False or
+        not set in the plist at all.
+
+    :rtype: bool
+
+    .. versionadded::
```

Comment:
Oops, this line should be `.. versionadded:: Fluorine`

Reply:
whoops, looks like you beat me to it. Thanks